cli/src/test/destroy.spec.ts

```diff
@@ -0,0 +1,63 @@
+import anyTest, { TestInterface } from 'ava';
+import * as td from 'testdouble';
+import { Context, addFake } from './fake_lycan';
+import { success } from 'specshell';
+import { readFile } from 'mz/fs';
+import { NotFoundError, UnauthorizedError } from '@binaris/spice-node-client/interfaces';
+import { safeLoad } from 'js-yaml';
+
+const test = anyTest as TestInterface<Context>;
+
+addFake(test);
+
+const anything = td.matchers.anything();
+
+test.beforeEach(async (t) => {
+  t.assert(success(await t.context.shell.run(`cd ${t.context.projectDir}`)));
+  t.log(t.context.projectDir);
+});
+
+test('no args in project dir', async (t) => {
+  td.when(t.context.lycanFake.destroyApp(anything, 'fluffy-samaritan')).thenResolve(null);
+  const result = await t.context.shell.run(`${t.context.run} destroy`, 'utf-8');
+  t.snapshot(result);
+  const config = await readFile(t.context.configPath, 'utf-8');
```

Comment:
Can you snapshot here, too?

Reply:
excellent suggestion. thanks. separates content from code.!